libs/island-ui/core/src/lib/ActionCard/ActionCard.tsx

```diff
@@ -130,16 +136,37 @@ export const ActionCard: React.FC<ActionCardProps> = ({
 
   const renderDefault = () => {
     const hasCTA = cta.label && !progressMeter.active
+    const hasSecondaryCTA =
+      hasCTA && secondaryCta?.label && !progressMeter.active
 
     return (
       <>
         {!date && renderTag()}
 
         {!!hasCTA && (
-          <Box paddingTop={tag.label ? 'gutter' : 0}>
-            <Button variant={cta.variant} size="small" onClick={cta.onClick}>
-              {cta.label}
-            </Button>
+          <Box
+            paddingTop={tag.label ? 'gutter' : 0}
+            display="flex"
+            justifyContent={['flexStart', 'flexEnd']}
+            alignItems="center"
+            flexDirection="row"
+          >
+            {hasSecondaryCTA && (
+              <Box paddingRight={4} paddingLeft={2}>
+                <Button
+                  variant="text"
+                  onClick={secondaryCta?.onClick}
+                  icon={'document'}
+                >
+                  {secondaryCta?.label}
```

Comment:
I think you should check if secondaryCta.label is present in line 154, i.e. `hasSecondaryCTA && secondaryCta.label &&...` because right now you could be rendering a button with no text.

Reply:
`hasSecondaryCTA` is created like this `const hasSecondaryCTA = hasCTA && secondaryCta?.label && !progressMeter.active` in line 139 so it is not rendered if label is empty. I tried setting `label: ''` and it works as expected :)